core/modules/parsers/wikiparser/rules/filteredtranscludeblock.js

```diff
@@ -7,44 +7,35 @@ Wiki text rule for block-level filtered transclusion. For example:
 
 ```
 {{{ [tag[docs]] }}}
-{{{ [tag[docs]] |tooltip}}}
 {{{ [tag[docs]] ||TemplateTitle}}}
-{{{ [tag[docs]] |tooltip||TemplateTitle}}}
-{{{ [tag[docs]] }}width:40;height:50;}.class.class
+((( [tag[docs]] )))
+((( [tag[docs]] ||TemplateTitle)))
```

Comment:
I am not happy about this one. There are 2 wikitext syntax variants now, that do the exact same thing.